File: minijax-examples/minijax-example-hello/src/main/java/com/example/Hello.java
package com.example;

import jakarta.ws.rs.GET;
import jakarta.ws.rs.Path;

import org.minijax.Minijax;

@Path("/")
public class Hello {

    @GET
    public static String hello() {
        return "Hello world!";
    }

    public static void main(String[] args) {
        new Minijax()
                .register(Hello.class)
                .start();
    }
}

File: minijax-examples/minijax-example-kotlin/src/main/kotlin/demo/Hello.kt
package demo

import org.minijax.Minijax
import javax.ws.rs.GET
import javax.ws.rs.Path

@Path("/")
class Hello {
    @GET
    fun get(): String = "Hello world!"
}

fun main(args : Array<String>) {
    Minijax().register(Hello::class.java).start()
}

